connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -51,25 +51,24 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   // Mapping from id to StreamingQueryListener. Used for methods like removeListener() in
   // StreamingQueryManager.
   private lazy val listenerCache: ConcurrentMap[String, StreamingQueryListener] =
-    new ConcurrentHashMap()
-
-  private[connect] def createExecutePlanHolder(
-      request: proto.ExecutePlanRequest): ExecutePlanHolder = {
+  new ConcurrentHashMap()
```

Comment:
```suggestion
    new ConcurrentHashMap()
```

Reply:
scalafmt failed so I made this fix.

Reply:
Thank you! I must have eaten this whitespace while resolving the conflict there...